src/main/java/com/SAS/Controllers/SASController/SASController.java

```diff
@@ -0,0 +1,60 @@
+/**
+ * This class represents the external controller of the system
+ */
+package com.SAS.Controllers.SASController;
+
+import com.SAS.LeagueManagement.LeagueManagementController;
+import com.SAS.User.User;
+import com.SAS.User.UserController;
+import com.SAS.User.UserType;
+import com.SAS.crudoperations.UsersCRUD;
+import com.SAS.teamManagenemt.TeamManagement;
+
+public class SASController {
+
+    private UserController userController;
+    private LeagueManagementController leaugeManagement;
+    private TeamManagement teamManagement;
+    private UsersCRUD usersCRUD;
+
+    /**
+     * Empty Constructor, initializes the other controllers
+     */
+    public SASController() {
```

Comment:
consider making this controller singleton.
SystemController need to be added also.

Reply:
It cannot be singleton since someone will need to hold the single instance and its exactly the goal, that it will hold all other controllers and no one will need to hold it..